components/dashboard/dashboard.css

```diff
@@ -46,83 +46,83 @@ body .dashboard-container #meeting-rooms .meeting-room-card {
   cursor: pointer;
   width: calc(50% - 20px);
 }
-body .dashboard-container #meeting-rooms .meeting-room-card .image-container {
+body .dashboard-container .meeting-rooms .meeting-room-card .image-container {
   position: relative;
   display: inline-block;
   width: 100%;
 }
-body .dashboard-container #meeting-rooms .meeting-room-card .image-container .card-image {
+body .dashboard-container .meeting-rooms .meeting-room-card .image-container .card-image {
   width: 100%;
   height: 200px;
   -o-object-fit: cover;
      object-fit: cover;
 }
-body .dashboard-container #meeting-rooms .meeting-room-card .image-container .room-status-container {
+body .dashboard-container .meeting-rooms .meeting-room-card .image-container .room-status-container {
   display: flex;
   justify-content: center;
   align-items: center;
   gap: 4px;
   position: absolute;
   bottom: 10px;
   left: 5px;
-  background-color: white;
+  background-color: #fdfeff;
   padding: 3px 10px;
-  color: #008f22;
+  color: #008d00;
```

Comment:
could this be also variable instead of directly using here like this ?

Reply:
This is the css mapped file, I would have changed in the original scss file